internal/distributed/proxy/httpserver/handler_v2.go

```diff
@@ -1159,8 +1185,39 @@ func (h *HandlersV2) createCollection(ctx context.Context, c *gin.Context, anyRe
 			Name:               httpReq.CollectionName,
 			AutoID:             httpReq.Schema.AutoId,
 			Fields:             []*schemapb.FieldSchema{},
+			Functions:          []*schemapb.FunctionSchema{},
 			EnableDynamicField: httpReq.Schema.EnableDynamicField,
 		}
+
+		allOutputFields := []string{}
+
+		for _, function := range httpReq.Schema.Functions {
+			functionTypeValue, ok := schemapb.FunctionType_value[function.FunctionType]
+			if !ok {
+				log.Ctx(ctx).Warn("function's data type is invalid(case sensitive).", zap.Any("function.DataType", function.FunctionType), zap.Any("function", function))
+				HTTPAbortReturn(c, http.StatusOK, gin.H{
+					HTTPReturnCode:    merr.Code(merr.ErrParameterInvalid),
+					HTTPReturnMessage: merr.ErrParameterInvalid.Error() + ", function data type " + function.FunctionType + " is invalid(case sensitive).",
+				})
+				return nil, merr.ErrParameterInvalid
```

Comment:
It is recommended not to throw err nakedly, but to use wrapErrXXX to carry necessary information.